src/data/associates_db.py

```diff
@@ -0,0 +1,69 @@
+''' File to define Associate MongoDB operations. '''
+import pymongo
+
+from src.data.data import _db
+
+from src.models.associates import Associate
+
+from src.logging.logger import get_logger
+
+_log = get_logger(__name__)
+
+_associates = _db['associates']
+
+def create_associate(new_associate: Associate):
+    '''Creates a new associate in the database'''
+    _associates.insert_one(new_associate.to_dict())
+
+def read_all_associates():
+    '''Returns all associates'''
+    return _associates.find({})
+
+def read_all_associates_by_query(query_dict):
+    '''Takes in a query_dict and returns a list of info based on that query'''
+    return list(_associates.find(query_dict))
+
+def read_one_associate_by_query(query_dict):
+    '''Takes in an associate query dict and returns one associate matching query.'''
+    return _associates.find_one(query_dict)
+
+def update_associate_swot(query_dict, swot_id):
+    '''Takes in a associate query_dict, a swot, and appends a swot_id'''
```

Comment:
Either this is incorrect or your definition is. I don't think this method does the right thing.